### PR TITLE
docs: Update mng-cluster-provider.md to remove typo

### PR DESCRIPTION
**Pull request by gdoteof**
*Issue #, if available:*

*Description of changes:*
It looks like a mass update to the readme failed here.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/cluster-providers/mng-cluster-provider.md [hunk 1]
```diff
@@ -9,7 +9,6 @@ const props: MngClusterProviderProps = {
     minSize: 1,
     maxSize: 10,
     desiredSize: 4,
-    version: "auto"
```
Comment (elamaran11) [inline docs/cluster-providers/mng-cluster-provider.md:12]: This line should not be removed. Rather `version: KubernetesVersion.V1_25,` line should be removed if you want to update it. Applies to other part of the doc too.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@gdoteof Thankyou for the PR. Please check our comments.

File: docs/cluster-providers/mng-cluster-provider.md (hunk 1 above)
Comment (shapirov103) [reply docs/cluster-providers/mng-cluster-provider.md:12]: +1 seems more reasonable not to hardcode the version

**Review by shapirov103 — COMMENTED** [review]
@gdoteof please address the comment, will merge after that.

**Comment by elamaran11** [comment]
@gdoteof Can you fix the above comment. We are planning to release soon!

**Review by shapirov103 — APPROVED** [review]
Upon further review the version is expect as KubernetesVersion so has to either be removed or type changed to eks.KubernetesVersion.
Approving,

**Review by shapirov103 — APPROVED** [review]

**Pull request closed without merging**